.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+### Prepare a Pull Request
+
+- Title: [BitSail-XYZ][XxxType] Title of the pull request
+
+- Fixes: #XYZ
```

Comment:
The advantage of this writing way is that GitHub will automatically associate the Issue to be processed by this PR.

When merging the current PR, the associated Issue will also be automatically closed.

![image](https://user-images.githubusercontent.com/31994335/223289414-c15eacd2-7425-48d5-b278-96132c949acb.png)